Use AWSSDKVersion variable for security token package

mungojam opened the pull request:
At the moment the version of the AWS security token package is 10 months out of date and I think that is why my attempt to use [credential_process](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html) isn't working.

It also seems like it was an oversight to use the variable for the S3 package but not this package.

emgarten commented:
These AWS packages may not share the same version based on when they were added to nuget.org, do you know if they are suppose to?

https://www.nuget.org/packages/AWSSDK.S3
https://www.nuget.org/packages/AWSSDK.SecurityToken

Feel free to update the AWS SDK in Sleet, I've been leaving it alone since it has been working, but if you are seeing issues then let's update it.

mungojam commented:
You're right, I think with the Java equivalent, they are consistent, or at least they were, but seeing the [.net changelog](https://github.com/aws/aws-sdk-net/blob/master/SDK.CHANGELOG.md) it's clear that they aren't kept consistent.

I've changed to a later, fixed version now which should resolve my issue

emgarten commented:
Thanks for the PR @mungojam!

emgarten commented:
This change is available in https://www.nuget.org/packages/Sleet/3.1.11
